crates/shared/src/sources/balancer_v2/pools/liquidity_bootstrapping.rs

```diff
@@ -1,4 +1,4 @@
-//! Module implementing two-token weighted pool specific indexing logic.
+//! Module implementing liquidity bootstrapping pool specific indexing logic.
```

Comment:
Trivial unrelated change that I snuck in.

Reply:
This is part of the previous PR and I also a suggestion comment to fix it there. Just FYI

Reply:
Merged your suggestion :smil